interface/modules/custom_modules/oe-module-weno/ModuleManagerListener.php

```diff
@@ -152,6 +152,8 @@ private function disable($modId, $currentActionStatus): mixed
         return $currentActionStatus;
```

Comment:
something seems very wrong about the above inferno stuff :)
(maybe you git repo is set to collect submodules? Byt default, it should not, but something is awry)